test/cases/koji.sh

```diff
@@ -122,9 +135,12 @@ sudo cp \
     /etc/pki/ca-trust/source/anchors/osbuild-composer-tests-ca-crt.pem
 sudo update-ca-trust
 
-greenprint "Restarting composer to pick up new config"
+start_jwt_auth_server
```

Comment:
I thought about this and had it so initially, but then we would have to source it, instead of executing it. We need to keep the PIDs somewhere, so that we can kill them later.